src/domain/community/contributor/ContributorsSearch/ContributorsSearchContainer.tsx

```diff
@@ -120,18 +144,25 @@ const ContributorsSearchContainer = ({
   });
 
   const organizations: PaginatedResult<OrganizationContributorFragment> = {
-    items: oragnizationsQueryResult.data?.organizationsPaginated.organization,
-    loading: oragnizationsQueryResult.loading,
-    hasMore: oragnizationsQueryResult.hasMore,
-    pageSize: oragnizationsQueryResult.pageSize,
-    firstPageSize: oragnizationsQueryResult.firstPageSize,
-    error: oragnizationsQueryResult.error,
-    fetchMore: oragnizationsQueryResult.fetchMore,
+    items: organizationsQueryResult.data?.organizationsPaginated.organization,
+    loading: organizationsQueryResult.loading,
+    hasMore: organizationsQueryResult.hasMore,
+    pageSize: organizationsQueryResult.pageSize,
+    firstPageSize: organizationsQueryResult.firstPageSize,
+    error: organizationsQueryResult.error,
+    fetchMore: organizationsQueryResult.fetchMore,
+  };
+
+  const { data: vcData, loading: loadingVCs } = useContributorsVirtualInLibraryQuery();
+
+  const virtualContributors = {
+    items: vcData?.platform.library.virtualContributors ?? [],
+    loading: loadingVCs,
   };
 
-  const loading = users.loading || organizations.loading;
+  const loading = users.loading || organizations.loading || virtualContributors.loading;
   const error = users.error || organizations.error;
-  return <>{children({ users, organizations }, { loading, error }, {})}</>;
+  return <>{children({ users, organizations, virtualContributors }, { loading, error }, {})}</>;
```

Comment:
This error handling is more for the pagination. As the VCs call currently doesn't support pagination, it's fine to be like that. In case of a query error with the initial load, the global error handler will notify the user.